templates/parachain/node/src/chain_spec.rs

```diff
@@ -18,7 +18,10 @@ pub struct Extensions {
 	pub relay_chain: String,
 	/// The id of the Parachain.
 	#[serde(alias = "paraId", alias = "ParaId")]
-	pub para_id: u32,
+	#[deprecated(
```

Comment:
TO remove the para_id from here actually. This node has an embedded runtime which has the new runtime API implemented, so no need to still support the chain spec para_id parsing.